app/code/Magento/Sales/Controller/Adminhtml/Order/Create/Reorder.php

```diff
@@ -90,10 +104,18 @@ public function execute()
             }
             $resultRedirect->setPath('sales/order/view', ['order_id' => $orderId]);
         } else {
-            $order->setReordered(true);
-            $this->_getSession()->setUseOldShippingMethod(true);
-            $this->_getOrderCreateModel()->initFromOrder($order);
-            $resultRedirect->setPath('sales/*');
+            try {
+                $order->setReordered(true);
+                $this->_getSession()->setUseOldShippingMethod(true);
+                $this->_getOrderCreateModel()->initFromOrder($order);
+                $resultRedirect->setPath('sales/*');
+            } catch (\Magento\Framework\Exception\LocalizedException $e) {
```

Comment:
Hi @cedmudit. I've noticed that we don't log the exception. It's not that critical in case of `LocalizedException` since we show the message using the `messageManager`. However, it's still recommended to log the exception.

In the case of catching `\Exception` it's very important to have the details in the error log. Otherwise, we just catch the exception and hide important error details. 

Please, add error logging for both cases. Thank you!

Reply:
Hi, @rogyar. I will add an error logging.